public-ui/src/app/data-browser/views/concept-chart/concept-chart-react.component.tsx

```diff
@@ -0,0 +1,407 @@
+import {
+  Component,
+  Input
+} from '@angular/core';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { AgeChartReactComponent } from 'app/data-browser/charts/chart-age/chart-age-react.component';
+import { BioSexChartReactComponent } from 'app/data-browser/charts/chart-biosex/chart-biosex-react.component';
+import { ValueReactChartComponent } from 'app/data-browser/charts/chart-measurement-values/chart-value-react.component';
+import { SourcesChartReactComponent } from 'app/data-browser/charts/chart-sources/chart-sources-react.component';
+import { TooltipReactComponent } from 'app/data-browser/components/tooltip/tooltip-react.component';
+import { ErrorMessageReactComponent } from 'app/data-browser/views/error-message/error-message-react.component';
+import { dataBrowserApi } from 'app/services/swagger-fetch-clients';
+import { reactStyles } from 'app/utils';
+import { GraphType } from 'app/utils/enum-defs';
+import { Spinner } from 'app/utils/spinner';
+import * as React from 'react';
+
+const styles = reactStyles({
+    sourceLayout: {
+        display: 'flex',
+        margin: '0 2rem',
+        height: '100%',
+        flexDirection: 'row'
+    },
+    sourcesChart: {
+      width: '100%',
+      marginLeft: '-1rem',
+      background: 'white'
+    },
+    conceptBoxInfo: {
+        padding: '.5rem',
+        textAlign: 'left'
+    },
+    measurementFilterChoice: {
+      color: '#262262'
+    },
+    ehrMChartLayout: {
+        display: 'flex',
+        flexWrap: 'wrap',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        paddingTop: '1em'
+    }
+});
+
+const cssStyles = `
```

Comment:
I see why we need `.unit-choice.active` and `.measurement-filter-choice.active` here, any reason not to move the other two classes to react styles?

Reply:
Changed this. thanks